cmd/notary-signer/config.go

```diff
@@ -68,7 +68,7 @@ func parseSignerConfig(configFilePath string, doBootstrap bool) (signer.Config,
 	}
 
 	// setup the cryptoservices
-	cryptoServices, err := setUpCryptoservices(config, []string{notary.MySQLBackend, notary.MemoryBackend, notary.RethinkDBBackend}, doBootstrap)
+	cryptoServices, err := setUpCryptoservices(config, []string{notary.MySQLBackend, notary.MemoryBackend, notary.PostgresBackend, notary.RethinkDBBackend}, doBootstrap)
```

Comment:
I wonder if we could introduce a constant to indicate those supported database backends in the [const file](https://github.com/docker/notary/blob/master/const.go), so that we could stop keeping inject new items into the arguments list if we support more backends in the future.

maybe something like
```
var NotarySupportedDB = []string{
	MemoryBackend,
	MySQLBackend,
	SQLiteBackend,
	RethinkDBBackend,
	PostgresBackend,
}

```